features/support/env.rb

```diff
@@ -41,7 +41,7 @@
 begin
   DatabaseCleaner.strategy = :transaction
 rescue NameError
-  raise "You need to add database_cleaner to your Gemfile (in the :test group) if you wish to use it."
+  raise 'You need to add database_cleaner to your Gemfile (in the :test group) if you wish to use it.'
```

Comment:
Not entirely. This is mostly the generated code but some modifications were made. Adding to the ignore list anyway on 4b853f1bb7735e3cb8ebd895327f5a9d107bf8ec